crates/jrsonnet-stdlib/Cargo.toml

```diff
@@ -16,9 +16,13 @@ exp-preserve-order = ["jrsonnet-evaluator/exp-preserve-order"]
 # Bigint type
 exp-bigint = ["dep:num-bigint", "jrsonnet-evaluator/exp-bigint"]
 
-exp-null-coaelse = ["jrsonnet-parser/exp-null-coaelse", "jrsonnet-evaluator/exp-null-coaelse"]
+exp-null-coaelse = [
+    "jrsonnet-parser/exp-null-coaelse",
+    "jrsonnet-evaluator/exp-null-coaelse",
+]
 # std.regexMatch and other helpers
 exp-regex = ["dep:regex", "dep:lru", "dep:rustc-hash"]
+yaml = ["dep:serde_yaml_with_quirks"]
```

Comment:
Yaml should be added to default-features,
```toml
default = ["yaml"]
```

But then this feature would need to be forwarded thru jrsonnet-cli/jrsonnet crates, and there is no easy way to opt-out of the features other than default-features=false, making this flag not easily usable.

Reply:
what is your suggestion?

Reply:
It is hard to make proper opt-out features, so I would prefer not to make yaml optional.